examples/platform/silabs/BaseApplication.cpp

```diff
@@ -300,6 +300,29 @@ CHIP_ERROR BaseApplication::StartAppTask(osThreadFunc_t taskFunction)
 }
 
 CHIP_ERROR BaseApplication::Init()
+{
+    CHIP_ERROR err = BaseInit();
+    if (err != CHIP_NO_ERROR)
+    {
+        SILABS_LOG("BaseInit() failed");
+        appError(err);
+        return err;
+    }
+
+    err = AppInit();
+    if (err != CHIP_NO_ERROR)
+    {
+        SILABS_LOG("AppInit() failed");
+        appError(err);
+        return err;
+    }
+
+    SILABS_TRACE_END_ERROR(TimeTraceOperation::kAppInit, err);
+    SILABS_TRACE_END_ERROR(TimeTraceOperation::kBootup, err);
```

Comment:
can you bring this to main?

Reply:
I thought since main = csa it didn't have the tracing

Reply:
will do.